utilities/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - python
     - iris
     - lxml
+    - owslib
+    - requests
```

Comment:
It might be a bit of work yes, but could be worthwhile. What interface did you have in mind?